.github/workflows/test_installer.yml

```diff
@@ -0,0 +1,39 @@
+name: Install Local Version of Opik
+
+on:
+    workflow_dispatch:
+
+jobs:
+    test_installation:
+        runs-on: ubuntu-latest
```

Comment:
I would hard code the version to our supported installer version

Reply:
What version is that?

Reply:
lets start with 20.04 LTS ad min supported.

Reply:
updated